src/components/Navbar/Navbar.js

```diff
@@ -58,7 +58,7 @@ class Navbar extends Component {
               <Menu
                 theme="dark"
                 mode="horizontal"
-                style={{ lineHeight: "64px" }}
+                style={{ lineHeight: "60px" }}
```

Comment:
@ritwizsinha Make the same change in the menu style if the user is not logged in